.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+     "cSpell.words": [
+          "Fjallraven",
+          "Foldsack"
+     ]
+}
```

Comment:
It's a good practice to not push this kind of configurations in gut.

you can create a `.gitignore` file and add this line to it:


```.gitignore
.vscode/
```

by this approach, any file in the .vscode directory will be ignored by the git